open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/server/mappers/classifications/ClassificationMapper.java

```diff
@@ -106,10 +108,18 @@ public Classification mapBeanToOmrs(org.odpi.openmetadata.accessservices.subject
          org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.Classification omrsClassification = new org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.Classification();
          SubjectAreaUtils.populateSystemAttributesForInstanceAuditHeader(systemAttributes, omrsClassification);
          // copy over the classification name
-         omrsClassification.setName(omasClassification.getClassificationName());
+         String classificationName = omasClassification.getClassificationName();
+         omrsClassification.setName(classificationName);
+         // copy over the classification type
+         OpenMetadataTypesArchiveAccessor archiveAccessor = OpenMetadataTypesArchiveAccessor.getInstance();
+         String classificationTypeGuid = archiveAccessor.getClassificationDefByName(classificationName).getGUID();
```

Comment:
Could you use the RepositoryHelper's getSkeletonClassification method here instea? It looks like it could save quite a bit of this construction code.

Reply:
@grahamwallis raised issue https://github.com/odpi/egeria/issues/3477 to track this suggested refactor.